docs/src/components/HomepageFeatures/index.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2024 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V.
+ *
+ * This work is licensed under the Fraunhofer License (on the basis of the MIT license)
+ * that can be found in the LICENSE file.
+ *
+ * The initial version of this file was auto-generated by Docusaurus. Docusaurus is published under
+ * the MIT license.
+ * See https://github.com/facebook/docusaurus/blob/6c7f2fde20b2448a1b966dd5f674205c0a08eeba/LICENSE
+ * for more information.
+ */
+import clsx from 'clsx';
+import Heading from '@theme/Heading';
+import styles from './styles.module.css';
+
+const FeatureList = [
+  {
+    title: 'Watermarker Library',
+    Svg: require('@site/static/img/library_undraw_software_engineer.svg').default,
+    description: (
+      <>
+        The main part of the TREND project. A library written in Kotlin, ready for
+        Kotlin, Java, and JavaScript applications.
+      </>
+    ),
+  },
+  {
+    title: 'Command Line Interface',
+    Svg: require('@site/static/img/cli_undraw_programmer.svg').default,
+    description: (
+      <>
+        A Command Line Interface (CLI Tool) for watermarking directly in a command
+        line. It uses the watermarker library.
+      </>
+    ),
+  },
+  {
+    title: 'Webinterface',
+    Svg: require('@site/static/img/webinterface_undraw_website.svg').default,
+    description: (
+      <>
+        A frontend to embed watermarks directly in a browser. It uses the watermarker
+        library.
```

Comment:
I think this wording is not correct. It means that we embed watermarks into browsers.
Perhaps this is better wording:
"A frontend for embedding watermarks directly from a browser. It uses the watermarker library."